src/styles/index.scss

```diff
@@ -1,3 +1,19 @@
-body {
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+@import "./utils/variables";
+@import "./utils/mixins";
+@import "./utils/fragments";
+
+@import "./blocks/card.scss";
+@import "./blocks/stars.scss";
+
+* {
+  font-family: Roboto, sans-serif;
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+li {
+  list-style-type: none;
 }
```

Comment:
Consider moving general page styles to a separate file to keep your main file clean and clear.